### ISSUE TITLE
Renpy launcher doesnt display the fonts correctly

### ISSUE THREAD

**Issue by reisxd** [renpy/renpy#2847]
I installed renpy with 7z exe and launcher wasnt displaying the fonts correctly. (I tried with sdk zip using winrar but no luck.)

Screenshot:
![image](https://user-images.githubusercontent.com/29177546/120765559-021dcf00-c522-11eb-9f63-e1da91553815.png)

**Comment by renpytom** [comment]
Can you please post your log.txt? I suspect that this is old video drivers, but I'd like to make sure.

**Comment by loafylemon** [comment]
I have received one report mentioning this issue as well. The user reported that the fonts were broken on GL2 renderer, but when they swapped to Angle2 renderer, the fonts went back to normal.

Here's the log file they attached:

# log.txt
```Sun May 30 10:29:47 2021
Windows-7-6.1.7601-SP1
Ren'Py 7.4.5.1648
 

Bootstrap to the start of init.init took 0.11s
Early init took 0.00s
Loader init took 0.25s
Loading error handling took 0.27s
Loading script took 3.18s
Loading save slot metadata. took 0.21s
Loading persistent took 0.01s
Importing _renpysteam: ImportError('No module named _renpysteam',)
Set script version to: (7, 4, 5)
Running init code took 0.18s
Loading analysis data took 0.13s
Analyze and compile ATL took 0.18s
Index archives took 0.00s
Dump and make backups. took 0.00s
Cleaning cache took 0.00s
Making clean stores took 0.00s
Initial gc. took 0.24s
DPI scale factor: 1.250000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: "Couldn't load nvlib." (can be ignored)
Creating interface object took 0.00s
Cleaning stores took 0.00s
Init translation took 0.24s
Build styles took 0.01s
Load screen analysis took 0.10s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.26s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Performance test:
Interface start took 0.89s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1920, 1080)
swap interval: 1 frames
Windowed mode.
Vendor: 'ATI Technologies Inc.'
Renderer: 'AMD Radeon HD 7570'
Version: '4.2.11627 Compatibility Profile Context'
Display Info: None
Screen sizes: virtual=(1080, 600) physical=(1330, 739) drawable=(1330, 739)
Could not open 'cache/shaders.txt':
Maximum texture size: 4096x4096
Total time until interface ready: 6.69038200378s
Screen sizes: virtual=(1080, 600) physical=(942, 987) drawable=(942, 987)
Could not open 'cache/shaders.txt':
Maximum texture size: 4096x4096
Screen sizes: virtual=(1080, 600) physical=(1330, 739) drawable=(1330, 739)
Could not open 'cache/shaders.txt':
Maximum texture size: 4096x4096```

**Comment by reisxd** [comment]
Sorry for being late and heres the log.txt:
```
Windows-10-10.0.19041
Ren'Py 7.4.5.1648
 

Bootstrap to the start of init.init took 0.21s
Early init took 0.01s
Loader init took 0.08s
Loading error handling took 0.04s
Loading script took 1.02s
Loading save slot metadata. took 0.01s
Loading persistent took 0.00s
Importing _renpysteam: ImportError('No module named _renpysteam',)
Set script version to: None (alternate path)
 - Init at launcher/game/web.rpyc:26 took 0.51100 s.
Running init code took 1.16s
Loading analysis data took 0.07s
Analyze and compile ATL took 0.01s
Index archives took 0.00s
Dump and make backups. took 0.00s
Cleaning cache took 0.00s
Making clean stores took 0.00s
Initial gc. took 0.15s
DPI scale factor: 1.000000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: "Couldn't load nvlib." (can be ignored)
Creating interface object took 0.02s
Cleaning stores took 0.00s
Init translation took 0.12s
Build styles took 0.01s
Load screen analysis took 0.06s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.19s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Performance test:
Interface start took 0.11s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1360, 1024)
swap interval: 1 frames
Windowed mode.
Vendor: 'ATI Technologies Inc.'
Renderer: 'ATI Radeon HD 4300/4500 Series'
Version: '3.3.11672 Compatibility Profile Context'
Display Info: None
Screen sizes: virtual=(800, 600) physical=(800, 600) drawable=(800, 600)
Maximum texture size: 4096x4096
Total time until interface ready: 4.68600106239s
```
Also, why is it a driver issue?

**Comment by rex-nihilo** [comment]
Same issue. Here's the log :

```
Windows-7-6.1.7601-SP1
Ren'Py 7.4.6.1693
 

Bootstrap to the start of init.init took 0.18s
Early init took 0.00s
Loader init took 0.02s
Loading error handling took 0.14s
Loading script took 0.71s
Loading save slot metadata. took 0.02s
Loading persistent took 0.00s
Importing _renpysteam: ImportError('No module named _renpysteam',)
Set script version to: None (alternate path)
Running init code took 0.60s
Loading analysis data took 0.05s
Analyze and compile ATL took 0.01s
Index archives took 0.00s
Dump and make backups. took 0.00s
Cleaning cache took 0.00s
Making clean stores took 0.00s
Initial gc. took 0.09s
DPI scale factor: 1.000000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: "Couldn't load nvlib." (can be ignored)
Creating interface object took 0.00s
Cleaning stores took 0.00s
Init translation took 0.08s
Build styles took 0.00s
Load screen analysis took 0.05s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.16s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Performance test:
Interface start took 0.10s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1920, 1080)
swap interval: 1 frames
Windowed mode.
Vendor: 'ATI Technologies Inc.'
Renderer: 'ATI Radeon HD 5570'
Version: '4.2.12002 Compatibility Profile Context 9.12.0.0'
Display Info: None
Screen sizes: virtual=(800, 600) physical=(835, 626) drawable=(835, 626)
Maximum texture size: 4096x4096
Total time until interface ready: 6.10834884644s
```

Problem with ATI Radeon ?

**Comment by renpytom** [comment]
It seems like it. It looks like there are newer drivers for that card, so I'd consider updating to them, in the hopes that it's just a bug in ATI's drivers.

**Comment by renpytom** [comment]
I purchased a Radeon HD 5450, and I'm unable to repeat this problem with the drivers on AMD's website. I made a pair of changes to fix this 'in the blind', but fundamentally the solution should probably be to upgrade to more modern hardware - all of these problems so far have had some combination of 'unsupported os' or 'gpu unsupported on os'.

**Comment by reisxd** [comment]
> I purchased a Radeon HD 5450, and I'm unable to repeat this problem with the drivers on AMD's website. I made a pair of changes to fix this 'in the blind', but fundamentally the solution should probably be to upgrade to more modern hardware - all of these problems so far have had some combination of 'unsupported os' or 'gpu unsupported on os'.
Yeah, it seems like its been fixed! Im closing the issue now.

**Comment by rex-nihilo** [comment]
Yes! Fixed on Renpy version 7.4.7
